test/e2e/tests/metrics/state-snapshots/errors-after-init-opt-in-ui-state.json

```diff
@@ -23,6 +23,7 @@
     "externalServicesOnboardingToggleState": "boolean",
     "keyringRemovalSnapModal": "object",
     "showKeyringRemovalSnapModal": "boolean",
+    "showNewSrpAddedToast": "boolean",
```

Comment:
This addition looks like it isn't supposed to be here. This key is already present below